src/core/api/public_api.ts

```diff
@@ -2142,6 +2167,52 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     }
   }
 
+  /**
+   * Enable the trickMode for playback.
+   * Select and set the trickMode track for current video track.
+   * Modify the trickMode playback rate.
+   * @param {number} playbackRate
+   */
+  setTrickMode(playbackRate: number): void {
+    if (this._priv_trickModeEnabled$.getValue()) {
+      return;
+    }
+    if (playbackRate === 0) {
+      throw new Error("API: Can't set trickMode for still playback (playback rate = 0)");
+    }
+    if (this._priv_trackChoiceManager === null ||
+        isNullOrUndefined(this._priv_contentInfos)) {
+      throw new Error("API: Can't set trickMode when no content loaded.");
+    }
+    const { currentPeriod } = this._priv_contentInfos;
+    if (currentPeriod === null) {
+      throw new Error("API: Can't set trickMode when no content loaded.");
+    }
+    this._priv_trickModeEnabled$.next(true);
+    this._priv_trackChoiceManager.setVideoTrickMode(currentPeriod);
```

Comment:
From what I see, this function can throw, in which case the trickmode speed won't be updated.